### PR TITLE
[RW-4480][Risk=no Add details for workspace libraries

### PR DESCRIPTION
**Pull request by blrubenstein**
Description:
Screen Shot 2020-03-04 at 2 23 29 PM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/workspace/workspace-library.tsx [hunk 1]
```diff
@@ -60,6 +60,9 @@ const libraryTabs = {
   },
   PHENOTYPE_LIBRARY: {
     title: 'Phenotype Library',
+    description: <div>These workspaces demonstrate how computable electronic phenotypes can be implemented within
+      the <i>All of Us</i> data using examples of previously published phenotype algorithms. You can open the
```
Comment (ericsong) [inline ui/src/app/pages/workspace/workspace-library.tsx]: "with All of Us data **set**"

or maybe

"can be implemented with All of Us data"

File: ui/src/app/pages/workspace/workspace-library.tsx [hunk 2]
```diff
@@ -70,6 +73,9 @@ const libraryTabs = {
   },
   TUTORIAL_WORKSPACES: {
     title: 'Tutorial Workspaces',
+    description: <div>These workspaces provide instructions for key Researcher Workbench components and representation
```
Comment (ericsong) [inline ui/src/app/pages/workspace/workspace-library.tsx]: representation**s**?

File: ui/src/app/pages/workspace/workspace-library.tsx [hunk 3]
```diff
@@ -70,6 +73,9 @@ const libraryTabs = {
...
+      of <i>All of Us</i> data. You can open the workspaces to view them or “duplicate” the workspaces to edit and
+      execute.</div>,
```
Comment (ericsong) [inline ui/src/app/pages/workspace/workspace-library.tsx]: execute them?

File: ui/src/app/pages/workspace/workspace-library.tsx [hunk 4]
```diff
@@ -97,9 +103,10 @@ interface ReloadableProfile {
 }
 
 interface CurrentTab {
-  title: string;
-  icon: string;
+  description?: string | JSX.Element;
```
Comment (ericsong) [inline ui/src/app/pages/workspace/workspace-library.tsx]: can we just make this `JSX.Element`?

Comment (blrubenstein) [reply ui/src/app/pages/workspace/workspace-library.tsx]: I think I am fine with that, but I want the option to use a string to be there. We can leave it out for now and add that eventually.

**Pull request merged**